src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -19,7 +18,7 @@ const DisplayAttributeInputs = ({
   attributeType,
   onInputAdd,
 }: DisplayAttributeInputsProps) => {
-  const [attributeValue, setAttributeValue] = useState();
+  const [attributeValue, setAttributeValue] = useState(null);
 
   const plusClick = (e) => {
```

Comment:
Probably `handleAttributeChange` would be more appropriate. Although not part of your task, seems quick change.

Reply:
I can change it no issues

Reply:
thank you!